### DISCUSSION TITLE
Table with Mixin class does not reflect in piccolo-admin

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo#614] [General]
I have tried to use mixin as stated in the [documentation](https://piccolo-orm.readthedocs.io/en/latest/piccolo/schema/advanced.html#mixins) to  reduce the amount of repetition, but the table is not visible in the `piccolo admin`. 

for eg:- 
If following is my Table, then `Manager` table is not visible in `piccolo admin`.
```python
from piccolo.columns import Varchar, Boolean
from piccolo.table import Table


class FavouriteMixin:
    favourite = Boolean(default=False)


class Manager(FavouriteMixin, Table):
    name = Varchar()
```

**Comment by dantownsend** [comment]
That's strange. Is `Manager` definitely being passed to `create_admin`?

I created a quick test app, and it seemed to be OK:

[admin_missing_table.zip](https://github.com/piccolo-orm/piccolo/files/9505465/admin_missing_table.zip)

Maybe there's a weird bug somewhere.

**Reply by sumitsharansatsangi** [reply]
True, Forget to pass the admin.

**Reply by dantownsend** [reply]
Cool - glad that's sorted. I've forgotten to do that before.
